Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -33,19 +33,19 @@ final class RemoteDataServicesManager {
     
     private var carbStore: CarbStore
 
-    private var doseStore: DoseStore
+    private var doseStore: DoseStoreProtocol
```

Comment:
Adding store protocols so the stores can be mocked to test LDM

Reply:
I don't understand why this can't keep using the normal stores?  You don't need to instantiate RemoteDataServicesManager to run LDM tests, right?

Reply:
Whoops, I meant to put that comment in LDM. Agreed that this should use `DoseStore`